Source/LibationWinForms/Form1.cs

```diff
@@ -12,24 +12,10 @@ namespace LibationWinForms
 {
 	public partial class Form1 : Form
 	{
-		private ProductsGrid productsGrid { get; }
-
 		public Form1()
 		{
 			InitializeComponent();
 
-			if (this.DesignMode)
```

Comment:
From my research, this will never be true inside a constructor.

Reply:
ctor and Load are called in DesignMode for inherited forms. To be consistent, either the ctor check can be removed or I should add defensive checks on each Load subscription

Reply:
Looks like this was the only instance of DesignMode being checked in a Form constructor. Everywhere else it's only checked in the Load event handler.